mackerel-plugin-openldap/lib/openldap.go

```diff
@@ -0,0 +1,355 @@
+				{Name: "bind_monitorOpInitiated", Label: "Bind", Diff: true},
+				{Name: "unbind_monitorOpInitiated", Label: "Unbind", Diff: true},
+				{Name: "search_monitorOpInitiated", Label: "Search", Diff: true},
+				{Name: "compare_monitorOpInitiated", Label: "Compare", Diff: true},
+				{Name: "modify_monitorOpInitiated", Label: "Modify", Diff: true},
+				{Name: "modrdn_monitorOpInitiated", Label: "Modrdn", Diff: true},
+				{Name: "add_monitorOpInitiated", Label: "Add", Diff: true},
+				{Name: "delete_monitorOpInitiated", Label: "Delete", Diff: true},
+				{Name: "abandon_monitorOpInitiated", Label: "Abandon", Diff: true},
+				{Name: "extended_monitorOpInitiated", Label: "Extended", Diff: true},
+			},
+		},
+		"operations_Completed": {
+			Label: (labelPrefix + " operations Completed"),
+			Unit:  "integer",
+			Metrics: []mp.Metrics{
+				{Name: "operations_monitorOpCompleted", Label: "All", Diff: true},
+				{Name: "bind_monitorOpCompleted", Label: "Bind", Diff: true},
+				{Name: "unbind_monitorOpCompleted", Label: "Unbind", Diff: true},
+				{Name: "search_monitorOpCompleted", Label: "Search", Diff: true},
+				{Name: "compare_monitorOpCompleted", Label: "Compare", Diff: true},
+				{Name: "modify_monitorOpCompleted", Label: "Modify", Diff: true},
+				{Name: "modrdn_monitorOpCompleted", Label: "Modrdn", Diff: true},
+				{Name: "add_monitorOpCompleted", Label: "Add", Diff: true},
+				{Name: "delete_monitorOpCompleted", Label: "Delete", Diff: true},
+				{Name: "abandon_monitorOpCompleted", Label: "Abandon", Diff: true},
+				{Name: "extended_monitorOpCompleted", Label: "Extended", Diff: true},
+			},
+		},
+		"waiters": {
+			Label: (labelPrefix + " waiters"),
+			Unit:  "integer",
+			Metrics: []mp.Metrics{
+				{Name: "waiters_read_monitorCounter", Label: "read", Diff: false},
+				{Name: "waiters_write_monitorCounter", Label: "write", Diff: false},
+			},
+		},
+		"threads": {
+			Label: (labelPrefix + " threads"),
+			Unit:  "integer",
+			Metrics: []mp.Metrics{
+				{Name: "threads_max_monitoredInfo", Label: "max", Diff: false},
+				{Name: "threads_open_monitoredInfo", Label: "open", Diff: false},
+				{Name: "threads_active_monitoredInfo", Label: "active", Diff: false},
+			},
+		},
+		"statistics": {
+			Label: (labelPrefix + " statistics"),
+			Unit:  "integer",
+			Metrics: []mp.Metrics{
+				{Name: "statistics_bytes_monitorCounter", Label: "bytes", Diff: true},
+				{Name: "statistics_pdu_monitorCounter", Label: "pdu", Diff: true},
+				{Name: "statistics_entries_monitorCounter", Label: "entries", Diff: true},
+				{Name: "statistics_referrals_monitorCounter", Label: "referrals", Diff: true},
+			},
+		},
+		"connections": {
+			Label: (labelPrefix + " connections"),
+			Unit:  "integer",
+			Metrics: []mp.Metrics{
+				{Name: "connections_total_monitorCounter", Label: "total connection", Diff: true},
+				{Name: "connections_current_monitorCounter", Label: "current connection", Diff: true},
```

Comment:
It looks like the value of `cn=Current,cn=Connections,cn=Monitor` is just current value, so `connections_current_monitorCounter` should be set to `Diff: false`. Or am I misunderstanding?